SIL.Archiving/Extensions.cs

```diff
@@ -112,13 +112,17 @@ public static void AppendDelimiter(this StringBuilder sb, string value, string d
 		public static string ToLatinOnly(this string s, string charReplacement, string spaceReplacement, string charsToIgnore)
```

Comment:
You could adjust the comment on the method to mention the single '.' feature in <remarks>